susemanager-utils/testing/automation/push-to-obs.sh

```diff
@@ -59,8 +63,20 @@ if [ ! -f ${CREDENTIALS} ]; then
 fi
 
 INITIAL_CMD="/manager/susemanager-utils/testing/automation/initial-objects.sh"
-CMD="/manager/susemanager-utils/testing/docker/scripts/push-to-obs.sh -d '${DESTINATIONS}' -c /tmp/.oscrc -p '${PACKAGES}' ${VERBOSE} ${TEST} ${OBS_TEST_PROJECT} ${EXTRA_OPTS}"
 CHOWN_CMD="/manager/susemanager-utils/testing/automation/chown-objects.sh $(id -u) $(id -g)"
-
 docker pull $REGISTRY/$PUSH2OBS_CONTAINER
-docker run --rm=true -v "$GITROOT:/manager" -v "/srv/mirror:/srv/mirror" --mount type=bind,source=${CREDENTIALS},target=/tmp/.oscrc $REGISTRY/$PUSH2OBS_CONTAINER /bin/bash -c "${INITIAL_CMD}; ${CMD}; RET=\${?}; ${CHOWN_CMD} && exit \${RET}"
+
+test -n "$PACKAGES" || {
+  PACKAGES=$(ls "$GITROOT"/rel-eng/packages/ \
+          | grep -v -x -e heirloom-pkgtools -e rhnclient -e smartpm -e jabberd-selinux -e oracle-rhnsat-selinux -e oracle-selinux -e oracle-xe-selinux -e spacewalk-monitoring-selinux -e spacewalk-proxy-selinux -e spacewalk-selinux -e cx_Oracle -e apt-spacewalk -e perl-DBD-Oracle -e spacewalk-jpp-workaround)
+}
+echo "Building packages in parallel"
```

Comment:
You are echoing this, even with you are not building (and submitting) in parallel.
```suggestion
echo "Starting building and submission at $(date)"
```

Feel free to improve it if you really want to show when you are building in parallel. Easiest way is by modifying how you are managing DOCKER_CMD.:

```
x) DOCKER_BG="-d";;
```
and
```
if [ "${DOCKER_BG}" == "-d" ]; then
  echo "Starting building and submission (parallel) at $(date)"
else
  echo "Starting building and submission (serial) at $(date)"
fi
```
Of course you will need to take care of the other mentions to `DOCKER_CMD`.